arex-instrumentation/database/arex-database-mybatis3/src/main/java/io/arex/inst/database/mybatis3/ExecutorWrapper.java

```diff
@@ -82,6 +99,51 @@ private <U> U call(MappedStatement ms, Object o, BoundSql boundSql, ThrowingSupp
         return result;
     }
 
+    private void restoreKeyHolder(MappedStatement ms, DatabaseExtractor executor, Object o) {
+        try {
+            Reflector reflector = new Reflector(o.getClass());
+            String[] keyHolderList = StringUtils.split(executor.getKeyHolder(), KEYHOLDER_SEPARATOR);
+            String[] keyProperties = ms.getKeyProperties();
+
+            if (keyHolderList == null || keyProperties == null) {
```

Comment:
Check  `keyProperties == null`  before calling any code to avoid creating unnecessary instances,  such as 
>Reflector reflector = new Reflector(o.getClass());